crates/bindings-csharp/Codegen/Module.cs

```diff
@@ -155,11 +152,18 @@ public override Scope.Extensions ToExtensions()
             );
         }
 
+        // If it's a tagged enum, then attempting to generate table methods will result in more noisy errors than it's worth.
+        // We already reported useful diagnostic while parsing, and we generated what we could (type-level methods), so let's stop here.
+        if (Kind is TypeKind.Sum)
+        {
+            return extensions;
+        }
+
         var autoIncFields = Members
             .Where(f => f.Attrs.HasFlag(ColumnAttrs.AutoInc))
             .Select(f => f.Name);
 
-        var iTable = $"SpacetimeDB.Internal.ITable<{ShortName}>";
+        var iTable = $"SpacetimeDB.Internal.ITable<{FullName}>";
```

Comment:
Is the switch from Short to Full name related to diagnostics? The module SDK PR also does these, but this should merge fine I think.

Reply:
Yeah, otherwise it was emitting more residual errors (in the last test that checks for MSBuild noise). I don't remember which ones anymore, but I remember it was necessary to reduce the total count.

Reply:
Looks like this is not necessary anymore, probably because we have table views now which already use full name. Going to revert and see if there's anything else I can revert as well.